dts/arm/nordic/nrf_common.dtsi

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/i2c/i2c.h>
+#include <dt-bindings/gpio/gpio.h>
+
+#include <arm/nordic/override.dtsi>
+/*
+ * This value can be overridden at the board level or in an application specific
+ * dts/arm/nordic/override.dtsi file.
+ */
+#ifndef NRF_DEFAULT_IRQ_PRIORITY
+#define NRF_DEFAULT_IRQ_PRIORITY 1
```

Comment:
Is seems a bit weird that we start defining preprocessor macros in .dtsi headers.
Is this going to be an accepted policy?

@galak @mbolivar-nordic ^^

If yes, I am fine with this, too.

Reply:
It has precedent in Linux as well as Zephyr.  I think a case can be made that they should be `.h` files (even though I believe I formerly requested it be `.dtsi`), but IMO that should not be a blocker for this until it's changed uniformly in Zephyr.

Reply:
Whether it is .h or .dtsi is not a blocker, of course.
But I want a wider consensus that we will be accepting the use of pre-processor macros/directives in DTS.

Reply:
We've accepted the use of preprocessor directives in #27719, so in my mind it's only relevant for defines.  https://zephyrproject.slack.com/archives/C18PLHN8H/p1598280951131500 has a discussion that relates to another discussion; the conclusion there is that trying to make a `.h` file work for both code and devicetree is problematic.

But if you want a wider consensus I suggest opening an issue.  Nobody's going to find these review comments when trying to research how a decision was reached.

Reply:
Nope, I copied the DT maintainers, above, with which I seek consensus, no need to file an issue.

Reply:
I'm fine with it if @galak is.